misc/chrome_extension/content-script.js

```diff
@@ -1,93 +1,114 @@
 'use strict';
+window.onload = function () {
+	// your code 
 
-var port = chrome.runtime.connect();
-//global since it needs to persist beyond the function scope
-var elementsToHide = [];
-
-//Parses the XPath
-//
-//sample return
-//{type:"title", value:[element, element]}
-//
-//title: The string for the type
-//xpathString: the xpath to parse
-//getTrue: BOOL, if the value return is the text instead of the element
-var parseXPath = function (title, xpathString, getText) {
-	try {
-		let nodes = document.evaluate(xpathString, document);
-		let e, elements = [];
-
-		if (nodes.resultType === XPathResult.UNORDERED_NODE_ITERATOR_TYPE || nodes.resultType === XPathResult.ORDERED_NODE_ITERATOR_TYPE) {
-			while (e = nodes.iterateNext()) {
-				if (getText) {
-					elements.push(e.nodeValue);
-				}
-				else {
-					elements.push(e);
+	setTimeout(function () {
+		let jsonToSend = {};
+		jsonToSend.reload = 1;
+		chrome.runtime.sendMessage(jsonToSend);
+	}, 1);
+
+	var port = chrome.runtime.connect();
+	//global since it needs to persist beyond the function scope
+	var elementsToHide = [];
+	var prev;
```

Comment:
Please prefix these variables with __ (2 underscores) so we understand these are "globally" scoped.